src/validations/organization.schema.ts

```diff
@@ -1,6 +1,18 @@
 import { z } from "zod";
 
+const selectSchema = z.object({
```

Comment:
We also have it in `src/validations/manualRun.schema.ts`.
Put it in a `shared` folder and import from there.

Reply:
done